package.json

```diff
@@ -29,39 +29,38 @@
   },
   "homepage": "https://github.com/google-github-actions/deploy-cloudrun#readme",
   "dependencies": {
-    "@actions/core": "^1.5.0",
+    "@actions/core": "^1.6.0",
     "@actions/exec": "^1.1.0",
     "@actions/tool-cache": "^1.7.1",
-    "@google-github-actions/setup-cloud-sdk": "^0.1.1",
-    "@lifeomic/attempt": "^3.0.0",
-    "@types/tmp": "^0.2.1",
-    "fs": "0.0.2",
-    "googleapis": "^91.0.0",
-    "google-auth-library": "^7.10.1",
+    "@google-github-actions/setup-cloud-sdk": "^0.3.0",
+    "@lifeomic/attempt": "^3.0.1",
```

Comment:
I think some of these are no longer used.

Reply:
Removed. Also moved a few to devDeps, since they are only used int ests